server-mirror-git/src/main/java/com/linecorp/centraldogma/server/internal/mirror/AbstractGitMirror.java

```diff
@@ -257,7 +257,8 @@ void mirrorRemoteToLocal(
             changes.put(mirrorStatePath, Change.ofJsonUpsert(
                     mirrorStatePath, "{ \"sourceRevision\": \"" + headCommitId.name() + "\" }"));
             // Construct the log message and log.
-            summary = "Mirror " + abbrId + ", " + remoteRepoUri() + '#' + remoteBranch() +
+            summary = "Mirror " + abbrId + ", " + remoteRepoUri() +
+                    (remoteBranch() != null ? '#' + remoteBranch() : "") +
```

Comment:
Question) In this case, the head branch exists but the user just didn't specify the remote branch.
Would it make sense to use `headBranchRef` instead?

Reply:
Hi @jrhee17, this is just removing `#null` in the message in case the user doesn't specify the branch.
![image](https://github.com/line/centraldogma/assets/11583408/2f164675-d8fc-41a5-b696-3190a90c4b2c)

Reply:
Sure, but isn't a branch still used?

https://github.com/line/centraldogma/blob/20772a861bdf5ec4656c909bf7e8fe82b146a8be/server-mirror-git/src/main/java/com/linecorp/centraldogma/server/internal/mirror/AbstractGitMirror.java#L175

Reply:
@jrhee17 agree, we use `headBranchRef` will be more clear. Please check my latest commit 🙇